### ISSUE TITLE
Memory usage when querying

### ISSUE THREAD

**Issue by themantalope** [jina-ai/executor-elasticsearch-indexer#15]
Hi,

After indexing ~2000 documents (images and text) I'm getting OOM and timeout issues when searching using a `Flow`. I wanted to know if other people have had this issue, or if there is some setting that I should be aware of to avoid this.

**Comment by JoanFM** [comment]
Have you made sure that you are setting a healthy top k parameter?

**Comment by themantalope** [comment]
yes set `parameters={'top_k:5, 'limit':5}`. will send video of behavior in slack
